docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.8"
+name: irma_sms_issuer
+
+services:
+  # Irma issuer service
+  irmaserver:
+    image: ghcr.io/privacybydesign/irma:v0.13.1
+    container_name: irma_server
+    working_dir: /irmago
+    ports:
+      - 8088:8088
+    expose:
+      - 8088
+    networks:
+      - irma-net
+    entrypoint:
+      - "irma"
+      - "server"
+      - "--no-auth=false"
+      - "--requestors={\"irma_sms_issuer\":{\"auth_method\":\"publickey\",\"key_file\": \"/config/pk.pem\"} }"
+      - "--port=8088"
+      - "--jwt-privkey-file=/config/sk.pem"
+      - "--url=http://ip-address:8088" # Your localhost IP-address here. This is required for the app to be able to connect to the issuer
+    volumes:
+      - ./build/resources/main/:/config/
+
+  # Service that runs the SMS issuer webapp and api
+  irma_sms_issuer:
+      platform: linux/x86_64 
+      build: 
+        context: .
+        dockerfile: Dockerfile
+      container_name: irma_sms_issuer
+      environment:
+         - IRMA_CONF=/config/
+      volumes:
+        # Make keys and config.json available in the container
+        - ./build/resources/main/:/config/
+        - ./src/main/resources/config.json:/config/config.json
+        # Make config.js available in the container
+        - ./webapp/config.js:/usr/local/tomee/webapps/ROOT/assets/config.js:ro" 
+      ports:
+        - 8080:8080
+      expose:
+        - 8080
+      networks:
+        - irma-net
+
+# Docker Desktop for MacOS does not support exposing ports when using host networking. Therefore,
+# we have to use bridge networking and expose the ports manually.
+# https://github.com/docker/for-mac/issues/1031
+networks:
+  irma-net:
+    driver: bridge
```

Comment:
You don't use any features that require host networking, since you expose all ports in the regular way and you don't use any custom DNS aliases or something. Therefore, you can fully remove this bridge network.

Reply:
You're right, removed